images/runtime/dotnetcore/dotnetCoreVersions.txt

```diff
@@ -1,4 +1,4 @@
-1.0, microsoft/dotnet:1.0.14-runtime
-1.1, microsoft/dotnet:1.1.11-runtime
+1.0, mcr.microsoft.com/dotnet/core/runtime:1.0.15
+1.1, mcr.microsoft.com/dotnet/core/runtime:1.1.12
 2.0, microsoft/aspnetcore:2.0.9
-2.1, microsoft/dotnet:2.1.8-aspnetcore-runtime
\ No newline at end of file
+2.1, mcr.microsoft.com/dotnet/core/runtime:2.1.9
```

Comment:
Shouldn't this be "mcr.microsoft.com/dotnet/core/**aspnet**" ? Also seems like 2.1.10 is available now: https://hub.docker.com/_/microsoft-dotnet-core-aspnet/

Reply:
done !